grakn-graql/src/main/java/ai/grakn/graql/internal/pattern/VarImpl.java

```diff
@@ -76,4 +76,21 @@ public Var getVarName() {
     public String toString() {
         return "$" + getValue();
     }
+
+    @Override
+    public final boolean equals(Object o) {
```

Comment:
It was here before - you're just looking at a sub-set of changes! I reverted it back to how it was.

I don't think this custom `equals` implementation is a good idea, but fixing it is too much for this PR.